system/data-privacy.rst

```diff
@@ -129,9 +130,13 @@ Frequently Asked Questions
    because there is no way to safely determine
    if it actually describes the user in question.
 
-😵 **I deleted an user and can still see a message they sent!**
+😵 I deleted an user and can still see a message they sent!
    Tickets can only belong to a single customer,
    but may contain messages (“articles”) from many people.
    If you deleted a user but you’re still seeing articles they sent,
    don’t worry—those articles are for a ticket that belongs to someone else,
    and no longer contain any reference to the sender’s identity.
+
+🚮 I removed a customer, my reporting now is off!
+   When removing users and their ticket, all references are removed.
```

Comment:
"now my reporting is off" instead of "my reporting now is off".

"tickets" instead of "ticket" in row 141.